stores/metadata.go

```diff
@@ -1321,6 +1321,39 @@ func (s *SQLStore) CopyObject(ctx context.Context, srcBucket, dstBucket, srcPath
 	return
 }
 
+func (s *SQLStore) MarkSectorLost(ctx context.Context, hk types.PublicKey, root types.Hash256) error {
```

Comment:
I'm not sure we need the concept of a "lost sector" in our database. This deletes all sectors for given host key and root and updates the affected slabs accordingly. We can keep the whole notion of hosts potentially losing sectors in the `worker` and just decide there if a host can't find the desired sector thus meaning that he lost it, we delete it?